libs/product/testing/src/factories/configurable-product.factory.spec.ts

```diff
@@ -39,6 +39,10 @@ describe('Product | Testing | Factories | DaffConfigurableProductFactory', () =>
 			expect(result.configurableAttributes).toBeDefined();
 			expect(result.variants).toBeDefined();
 			expect(result.variants[0].in_stock).toBeDefined();
-    });
+		});
+		
+		it('should return any percents as whole numbers', () => {
+			expect(Math.floor(result.variants[0].discount.percent)).toEqual(result.variants[0].discount.percent);
```

Comment:
I think mod 1 may make this test more clear? Flooring to check for integerness seems strange.

Reply:
changed